src/main/java/Pebble/Storage.java

```diff
@@ -0,0 +1,61 @@
+package pebble;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+<<<<<<< Updated upstream
+ * 
+=======
+ * Class to handle storage of tasks list on local device.
+>>>>>>> Stashed changes
+ */
```

Comment:
Do take not of this. I believe there was some issue with you stashing changes and not committing them but I am not too sure.

Reply:
oh i have no idea that was there, thanks for flagging this to me!! 🙏